recipes/pro-mdnsd/all/conandata.yml

```diff
@@ -2,6 +2,11 @@ sources:
   "0.8.4":
     url: "https://github.com/Pro/mdnsd/archive/v0.8.4.tar.gz"
     sha256: "e2d740d59b71b1b4478471c71fe5c2c96eea9296d42385b6e6525526a487d158"
+  "0.8.4.1":
```

Comment:
This version seems to be tagged but not released, do you have any insight into whether upstream expects to mark it as an official release soon? (Given that 0.8.4 is 5 years old at this point)

Reply:
https://github.com/Pro/mdnsd/issues/36

Reply:
Done

Reply:
@AbrilRBS can we please merge this pr?